x-pack/plugins/ml/public/embeddables/types.ts

```diff
@@ -210,6 +262,7 @@ export interface AnomalyChartsCustomOutput {
   severity?: number;
   indexPatterns: DataView[];
 }
+// @TODO: DELETE
```

Comment:
Is this comment still relevant?

Reply:
Removed here https://github.com/elastic/kibana/pull/183456/commits/9f05570550c8ecb43f34f2bff098fd98b7bfbdd0